generators/app/templates/_dockerTaskGeneric.sh

```diff
@@ -18,7 +18,12 @@ buildImage () {
        ENVIRONMENT="debug"
     fi
 
-    dockerFileName="Dockerfile.$ENVIRONMENT"
+    dockerFileName=""
+    if [[ $ENVIRONMENT == "release" ]]; then
+        dockerFileName="dockerfile"
+    else
+        dockerFileName="dockerfile.$ENVIRONMENT"
```

Comment:
You could simplify this further by using release as the default and doing the if on if ($Environment -ne "Release") { as it was previously done below.

---
In reply to: [67450925](https://github.com/Microsoft/generator-docker/pull/67#discussion_r67450925) [](ancestors = 67450925)